cognite/neat/_graph/loaders/_rdf2dms.py

```diff
@@ -63,78 +61,38 @@ class DMSLoader(CDFLoader[dm.InstanceApply]):
 
     def __init__(
         self,
+        dms_rules: DMSRules,
+        info_rules: InformationRules,
         graph_store: NeatGraphStore,
-        data_model: dm.DataModel[dm.View] | None,
```

Comment:
I remember I made this independent of the rules originally. Which why it takes in a DataModel. With the new structure, that is no longer possible. Long way of saying good :)